test/unit/server/routes/health.spec.js

```diff
@@ -22,23 +28,33 @@ describe('GET /health', () => {
       }),
   );
 
-  context('when the build-info.json file is present', () => {
-    const projectRoot = path.resolve(__dirname, '../../../../');
-    const buildJson = path.resolve(projectRoot, 'build-info.json');
+  context('when the DB isn\'t working', () => {
```

Comment:
As the stubs are created for the test and injected, there's nothing to clean up.

What would you use instead of stubs? Mocks? I generally prefer to use spy-type assertions at the end of the tests, and/or assert on behaviour exhibited by the stubs. The idea being that the test still mostly reads top-to-bottom and isn't overly sensitive to the exact order/arguments being called.